src/panels/lovelace/common/generate-lovelace-config.ts

```diff
@@ -468,6 +468,28 @@ export const generateLovelaceConfigFromHass = async (
   hass: HomeAssistant,
   localize?: LocalizeFunc
 ): Promise<LovelaceConfig> => {
+  if (hass.config.state !== "RUNNING") {
+    return {
+      title: hass.config.location_name,
+      views: [
+        {
+          cards: [{ type: "starting" }],
```

Comment:
Should this be a card or use `hass-loading-screen` instead ? That seems more in line with how we do loading screens.

Reply:
Using `hass-loading-screen` will be a lot more work (we now just use Lovelace logic) and will not give a better experience I think?